<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pvgMainWindow</class>
 <widget class="QMainWindow" name="pvgMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="pqTabbedMultiViewWidget" name="multiViewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_Files"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>Sources</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSources"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="pqStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="propertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOutput">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqOutputWidget" name="outputWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetPipeline">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>pqStatusBar.h</header>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqTabbedMultiViewWidget</class>
   <extends>QWidget</extends>
   <header>pqTabbedMultiViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqOutputWidget</class>
   <extends>QWidget</extends>
   <header>pqOutputWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
